src/AppBundle/Entity/Club.php

```diff
@@ -55,98 +54,104 @@ public function __construct()
 
     /**
      * @ORM\OneToMany(targetEntity="AppBundle\Entity\User", mappedBy="club")
-     * @var Collection|User[]
      */
     private $users;
 
+    public function __construct()
+    {
+        $this->created_at = new \DateTime();
```

Comment:
I will suggest to pass \Datetime object in constructor during object creation. It will be easier to test. Also you should use \DatetimeImmutable because created_at should not change during object life cycle.